jenkins-jobs/es-cmssw-afs-eos.py

```diff
@@ -22,7 +22,7 @@
     continue
   week, rel_sec  = cmsswIB2Week (payload["release"])
   payload["@timestamp"]=rel_sec*1000
-  id = sha1("%s-%s-%s" % (payload["release"], payload["architecture"], payload["fstype"])).hexdigest()
+  id = sha1(("%s-%s-%s" % (payload["release"], payload["architecture"], payload["fstype"])).encode()).hexdigest()
```

Comment:
@iarspider , I think the `shebang` statement for this script should be update too ... right?

Reply:
Fixed